tools/role_permission_configurator.php

```diff
@@ -0,0 +1,516 @@
+<?php
+/**
+ * Script to add or remove permissions from a role
+ *
+ * PHP Version 5-7
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Tara Campbell <tara.campbell@gmail.com>
+ * @license  Loris license
+ * @link     https://www.github.com/aces/Loris
+ */
+
+require_once __DIR__ . "/../vendor/autoload.php";
+require_once __DIR__ . "/generic_includes.php";
+
+$roleObject       = new \Role($DB);
+$permissionObject = new \Permission($DB);
+$userObject       = new \User();
+
+$roles       = $roleObject->getRoles();
+$permissions = $permissionObject->getPermissions();
+
+//##################################################################################
+//
+// PARSING ARGUMENTS
+//
+//##################################################################################
+// User did not provide any args
+if (count($argv) === 1) {
+    syntaxIncorrect();
+    exit(1);
+}
+
+if (count($argv) === 2 && isset($argv[1])) {
+    switch ($argv[1]) {
+        case "roles":
+            // User asking for the list of roles
+            if (!empty($roles)) {
+                echo "The roles in the database are:\n\n";
+                prettyPrint($roles);
+            }
+            else {
+                echo "There are currently no roles in the database.\n";
+            }
+            exit(2);
+        case "permissions":
+            // User asking for the list of permissions
+
+            echo "The permissions in the database are:\n\n";
+            prettyPrint($permissions);
+            exit(3);
+        case "rebuildRoles":
```

Comment:
exit code other than 0 are normally for error or abnormal ending. When there is no error or problem, exit code is normally 0.

Could also use a single exit() call with a variable.